docs/cookbook/checkout.rst

```diff
@@ -64,23 +68,25 @@ Remove the ``shipment_selected`` state, ``select_shipment`` transition. Remove t
                         on: ["complete"]
                         do: ["@sm.callback.cascade_transition", "apply"]
                         args: ["object", "event", "'create'", "'sylius_order'"]
-                    sylius_hold_inventory:
+                    sylius_save_checkout_completion_date:
                         on: ["complete"]
-                        do: ["@sylius.inventory.order_inventory_operator", "hold"]
+                        do: ["object", "completeCheckout"]
                         args: ["object"]
-                    sylius_assign_token:
-                        on: ["complete"]
-                        do: ["@sylius.unique_id_based_order_token_assigner", "assignTokenValue"]
+                    sylius_skip_shipping:
```

Comment:
@CoderMaggie is it necessary to define this callback when we skip the shipping step? Tried it without this callback and it still worked!?